### ISSUE TITLE
How to rerun this project in ucf101?

### ISSUE THREAD

**Issue by sja9527** [MCG-NJU/TDN#56]
Hi!  Thanks your great job! But I do  not Know how to train on the dataset of ucf101? Can you offer more details?

**Comment by yztongzhan** [comment]
Hi @sja9527! To reproduce our results on UCF101, you need to prepare the pre-trained model and add this argument `--tune_from='checkpoint/best.pth.tar` to your command. We use the following script for fine-tuning:
```
python -m torch.distributed.launch --master_port 12347 --nproc_per_node=8 \
main.py hmdb51 RGB --arch resnet50  --num_segments 16 --gd 20 --lr 0.002 \
--lr_steps 10 20 --epochs 25 --batch-size 16 --dropout 0.8 --consensus_type=avg \
--eval-freq=1 -j 4 --tune_from='checkpoint/best.pth.tar' --npb 
```
Then we perform 10-clip and 3-crop for accuracy evaluation protocol following [TEA](https://arxiv.org/abs/2004.01398) and [STM](https://arxiv.org/abs/1908.02486).
We recommend that you modify your learning rate according to the [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)
